### ISSUE TITLE
Raspberry Pi OS 64bit

### ISSUE THREAD

**Issue by Tony763** [pi-top/pi-topd#170]
Hello, following [pi-top-and-raspberry-pi-os](https://knowledgebase.pi-top.com/knowledge/pi-top-and-raspberry-pi-os) to get support for pi-topd on Raspberry Pi OS 64bit won't work.

Packages from your repository have multiple dependency issues.

```shell
tony@RPI2 ~ [4] % sudo apt install --no-install-recommends -y pt-device-manager pt-sys-oled pt-firmware-updater           16:01:40
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 pt-firmware-updater : Depends: i2c-tools-extra but it is not installable
                       Depends: pt-notifications but it is not going to be installed
                       Depends: python3-pycrc but it is not going to be installed
 python3-pitop : Depends: python3-dlib (< 19.23.0) but it is not installable
                 Depends: python3-dlib (>= 19.22.0) but it is not installable
                 Depends: python3-flask (< 1.1) but 1.1.2-2 is to be installed
                 Depends: python3-gevent (< 1.4) but 20.9.0-2 is to be installed
                 Depends: python3-imutils (< 0.6.0) but it is not going to be installed
                 Depends: python3-imutils (>= 0.5.4) but it is not going to be installed
                 Depends: python3-matplotlib (< 3.1) but 3.3.4-1 is to be installed
                 Depends: python3-opencv (< 3.3) but it is not going to be installed
                 Depends: python3-opencv (>= 3.2.0) but it is not going to be installed
                 Depends: python3-scipy (< 1.2) but 1.6.0-2 is to be installed
                 Depends: python3-zmq (< 17.2.0) but 20.0.0-1+b1 is to be installed
 python3-pyftdi : Depends: python3-pyserial (>= 3.0) but it is not going to be installed
                  Depends: python3-pyusb (>= 1.0.0) but it is not going to be installed
                  Depends: python3.7 but it is not installable
 python3-smbus2 : Depends: python3.7 but it is not installable
E: Unable to correct problems, you have held broken packages.
tony@RPI2 ~ [100] % python --version                                                                                      16:04:13
Python 3.9.2
```

**Comment by Tony763** [comment]
Just found arm64 is not supported at all, any plans for it? Raspberry Pi OS 64bit is now officially supported.

**Comment by Tony763** [comment]
Hi @m-roberts, thank you for linking PR with arm64 builds. 

For now, I started to play with armhf build and clean Raspberry Pi OS. 
Could somebody update [KB](https://knowledgebase.pi-top.com/knowledge/pi-top-and-raspberry-pi-os) with actual repository links for bullseye? KB contain only instructions for sirius. I dumped repository link and GPG key from Pi-top OS, but It's not so convenient way.

`deb [signed-by=/usr/share/keyrings/pi-top-os.gpg] https://packages.pi-top.com/pi-top-os/debian/ bullseye main`

**Comment by jcapona** [comment]
hey @Tony763 - sorry for the late response... we've just released 64-bit packages to our apt repositories.

The [Knowledge Base article](https://knowledgebase.pi-top.com/knowledge/pi-top-and-raspberry-pi-os) has been updated with the steps to install our apt repository in Raspberry Pi OS and what needs to be done to provide device support.

Big thanks to @m-roberts for all the work in getting arm64 packages out!
